incubator/hnc/cloudbuild.yaml

```diff
@@ -39,7 +33,7 @@ steps:
   - '-H'
   - 'Content-Type: application/x-application'
   - '--data-binary'
-  - '@multi-tenancy/incubator/hnc/out/hnc-manager.yaml'
```

Comment:
Why was it using out here?

Reply:
It's replaced by the line below. The `out` directory was only used by the Cloud Build script; now that we're generating these files using `make`, the manifests are in the `manifests/` directory, just like when you're developing it yourself.